src/apolo_app_types/app_types.py

```diff
@@ -55,21 +56,25 @@ def is_appless(self) -> bool:
 
     @classmethod
     def from_app_inputs(cls, inputs: AppInputs) -> "AppType":
-        if isinstance(inputs, LLMInputs):
-            return AppType.LLMInference
-        if isinstance(inputs, WeaviateInputs):
-            return AppType.Weaviate
-        if isinstance(inputs, StableDiffusionInputs):
-            return AppType.StableDiffusion
-        if isinstance(inputs, DockerHubInputs):
-            return AppType.DockerHub
-        if isinstance(inputs, HuggingFaceCacheInputs):
-            return AppType.HuggingFaceCache
-        if isinstance(inputs, SparkJobInputs):
-            return AppType.SparkJob
-        if isinstance(inputs, MLFlowAppInputs):
-            return AppType.MLFlow
-        if isinstance(inputs, JupyterAppInputs):
-            return AppType.Jupyter
+        match inputs:
```

Comment:
nit, maybe

```suggestion
        match type(inputs):
        case LLMInputs:
        ...
```

Reply:
my bad.

in case of use bare types, Python tries to assign values, not compare types. your approach is right :+1: